ManatiCR/vitaminau
# VIT-2: Registration Form

# Task [VIT-2](https://manati.atlassian.net/browse/VIT-2?atlOrigin=eyJpIjoiZGFiM2UyMTg1YWJmNDIwZGE1ZTk2YTQ0OWQwZmQ5YjIiLCJwIjoiaiJ9)
Creation of the "Venta de productos" form.

#Installation
- Run the following command
`ahoy composer install`
`ahoy drush cr && ahoy drush cim -y`

# Test create new account
- Go to the following url: 
  - http://vitaminau.docker/user/register
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-2](https://manati.atlassian.net/browse/VIT-2?atlOrigin=eyJpIjoiZGFiM2UyMTg1YWJmNDIwZGE1ZTk2YTQ0OWQwZmQ5YjIiLCJwIjoiaiJ9)
- Fill in all the fields of the form and press the "Create new account" button
- Go to the following url: 
  - http://mailhog.vitaminau.docker/
- Look for the email called "Account details for "[NAME_YOUR_ACCOUNT]" at VitaminaU"
- Go to the address found in the sent email.
- Fill in the password, password confirmation fields and save it.

# Test create new account
- Go to the following url: 
  - http://vitaminau.docker/user/password
- Enter the username or email of the account you want to change the password.
- Go to the following url: 
  - http://mailhog.vitaminau.docker/
- Look for the email called "Login information replacement for "[NAME_YOUR_ACCOUNT]" in VitaminaU"
- Go to the address found in the sent email.
- Fill in the password, password confirmation fields and save it.

# Acceptance criteria
- The user accesses a url with a registration form
- The user completes his registration data and receives a confirmation email
- The user completes the registration with the link in the confirmation email
- The user can reset password
